docs/cloud/managing-airbyte-cloud/review-connection-timeline.md

```diff
@@ -2,53 +2,75 @@
+It also shows who took an action, if that's relevant to the event.
 
-:::
+## Syncs, refreshes, and clears
 
-On this page, you can also view the complete logs and find any relevant errors, find a link to the job to share with Support, or download a copy of the logs locally.
+Completed syncs, [refreshes](/operator-guides/refreshes), and [clears](/operator-guides/clear) have a status associated to the event.
+
+| Status     | Description                                                                                                                            |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Succeeded  | Airbyte extracted 100% of the data and loaded to the destination.                                                                      |
+| Incomplete | The sync encountered an error, and should resolve itself. Airbyte loaded a subset or none of the data to the destination.              |
+| Failed     | The sync encountered a fatal error, and needs intervention to resolve. Airbyte loaded a subset or none of the data to the destination. |
+| Cancelled  | Someone cancelled the sync before it finished.                                                                                          |
+| Running    | The sync is currently running.                                                                                                         |
+
+### How the Connection Timeline reflects moved data
+
+For sync and refresh events, Airbyte also shows the volume of synced data, such as the amount of data moved, the number of records read and committed, and the total sync time. Reviewing this timeline can help you monitor the connection and identify any potential issues.
+
+| Data                             | Description                                                    |
+| -------------------------------- | -------------------------------------------------------------- |
+| `x` GB (also measured in KB, MB) | Amount of data moved during the sync.                          |
+| `x` extracted records            | Number of records read from the source during the sync.        |
+| `x` loaded records               | Number of records the destination confirmed it received.        |
+| `xh xm xs`                       | Total time (hours, minutes, seconds) for the sync to complete. |
+
+### How the Connection Timeline reflects failed syncs
+
+Most syncs only contain a single attempt, but syncs can fail. In the event of a failure, Airbyte attempts to sync your data before waiting for the next scheduled sync. [Learn more about Airbyte's retry approach](../../understanding-airbyte/jobs.md#retry-rules). If Airbyte needs multiple attempts to sync, you can open the logs to see the individual log from each attempt.
```

Comment:
> Most syncs only contain a single attempt, but syncs can fail.
I don't quite see the relation of a single attempt to failure here, should we be saying something like "Most syncs succeed after only one attempt, but repeated failed attempts can cause the sync to fail."